istioctl/pkg/writer/pilot/status.go

```diff
@@ -176,8 +178,13 @@ func (s *XdsStatusWriter) setupStatusPrint(drs map[string]*xdsapi.DiscoveryRespo
 				}
 				cds, lds, eds, rds := getSyncStatus(&clientConfig)
 				cp := multixds.CpInfo(dr)
+				meta, err := model.ParseMetadata(clientConfig.GetNode().GetMetadata())
+				if err != nil {
+					return nil, nil, fmt.Errorf("could not parse node metadata: %w", err)
```

Comment:
Nope, looks like if not sent it will not be an error

Reply:
Yes it won't be an error. Maybe the next improvement can be unifying both of the two commands to show something like `unknown` instead of leaving the cluster field blank if not obtained.